packages/react/src/primitives/types/flex.ts

```diff
@@ -40,4 +41,31 @@ export interface FlexStyleProps {
   wrap?: ResponsiveStyle<Property.FlexWrap>;
 }
 
-export interface FlexProps extends FlexStyleProps, BaseComponentProps {}
+export interface FlexProps extends ViewProps, FlexContainerStyleProps {}
+
+export interface FlexItemStyleProps {
+  /**
+   * Controls order flex items appear
+   */
+  order?: ResponsiveStyle<Property.Order>;
+
+  /**
+   * Ability for flex item to grow
+   */
+  grow?: ResponsiveStyle<Property.FlexGrow>;
```

Comment:
Note that this `grow` prop conflicts with the Tabs `grow` prop. I opted to rename the Tabs `grow` prop to `spacing` since I thought the native Flex CSS property should have naming priority.